Makefile

```diff
@@ -17,48 +17,32 @@ SHELL := /bin/bash
 GO               = go
 TIMEOUT          = 15
 PKGS             = ./...
-TOOLS_MODULE_DIR = ./internal/tools
+BUILD_DIR        = ./build
 TEST_RESULTS     = $(CURDIR)/test-results
 
 # Verbose output
 V = 0
 Q = $(if $(filter 1,$V),,@)
 
-# ALL_MODULES includes ./* dirs (excludes . and ./internal/tools dir).
+# ALL_MODULES includes ./* dirs (excludes . and ./build dir).
 ALL_MODULES := $(shell find . -type f -name "go.mod" -exec dirname {} \; | sort )
 # All directories with go.mod files related to opentelemetry library. Used for building, testing and linting.
-ALL_GO_MOD_DIRS := $(filter-out $(TOOLS_MODULE_DIR), $(ALL_MODULES))
+ALL_GO_MOD_DIRS := $(filter-out $(BUILD_DIR), $(ALL_MODULES))
 # All directories sub-modules. Used for tagging and generating dependabot config.
 SUBMODULES = $(filter-out ., $(ALL_GO_MOD_DIRS))
 
-.DEFAULT_GOAL := all
-
-.PHONY: all
-all: mod-tidy build lint license-check test-race
-
-.PHONY: ci
-ci: mod-tidy build lint markdownlint license-check diff
+.DEFAULT_GOAL := goyek
+.PHONY: goyek
+goyek:
+	./goyek.sh
 
 .PHONY: build
 build: # build whole codebase
 	${call for-all-modules,$(GO) build $(PKGS)}
 # Compile all test code.
 	${call for-all-modules,$(GO) test -vet=off -run xxxxxMatchNothingxxxxx $(PKGS) >/dev/null}
 
-# Tools
-
-TOOLS = $(CURDIR)/.tools
-
-$(TOOLS):
-	@mkdir -p $@
-$(TOOLS)/%: | $(TOOLS)
-	$Q cd $(TOOLS_MODULE_DIR) \
-		&& $(GO) build -o $@ $(PACKAGE)
-
-GOLANGCI_LINT = $(TOOLS)/golangci-lint
```

Comment:
Can we add a target to install `goyek`?

Reply:
https://github.com/signalfx/go-pipeline/blob/427de9c7dae208613be08ba02995b97b68a75b71/pipeline.go#L257-L282